tkg/constants/featureflags.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package constants
+
```

Comment:
`tanzu-framework/tkg` because `cluster` and `management-cluster` plugin share most of the codebase. This package is an implementation of what both of the plugins does. As both plugins imports this dependency and both plugin uses this featureflags, this constants were added to the common library instead of plugin specific code.
Also, this library heavily uses this constants.

Reply:
Sounds good